source/contracts/reporting/Market.sol

```diff
@@ -324,8 +332,20 @@ contract Market is DelegationTarget, ITyped, Initializable, Ownable, IMarket {
         require(_forkingMarket != this);
         require(!isFinalized());
         IInitialReporter _initialParticipant = getInitialReporter();
+        // Early out if already disavowed or nothing to disavow
+        if (_initialParticipant.getReportTimestamp() == 0) {
+            return true;
+        }
         delete participants;
         participants.push(_initialParticipant);
+        // Send REP from the no show bond back to the address that placed it. If a report has been made tell the InitialReporter to return that REP and reset
+        IReputationToken _reputationToken = getReputationToken();
+        if (noShowBond > 0) {
```

Comment:
Note: If there's a no-show bond present on the market, no initial report has occurred, and we return it to the noShowBondOwner (who is either the original owner, or the noShowBondOwner if this market has already been migrated previously)).

If the initial report HAS occurred, and we are disavowing, send back the rep to the initial reporter (not necessary for the crowdsourcers, since they recognize their disavowal via non-membership to market)). This initialReporter will be re-used in the next universe